### PR TITLE
Bug 1119709 - [FxOs2.2][Settings]The security word is turned in Settings...

### PR DESCRIPTION
**Pull request by EragonJ**
...-> Internet Sharing

### CODE REVIEW

**Comment by try-server-hook** [comment]
[EragonJ EragonJ (E.J.) (EragonJ)](https://github.com/EragonJ) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=b50d93da4cee)

File: apps/settings/style/lists.css [hunk 1]
```diff
@@ -106,7 +106,27 @@ ul > li > progress + span {
 }
 
 /* description + value on the same line */
-ul li > a span:nth-of-type(2):not(.button) {
+ul li > .box-container .box-title {
```
Comment (crh0716) [inline apps/settings/style/lists.css]: How about info container? I don't have good idea, either. :(

**Comment by try-server-hook** [comment]
[EragonJ EragonJ (E.J.) (EragonJ)](https://github.com/EragonJ) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=a48725f48b08)

**Comment by try-server-hook** [comment]
[EragonJ EragonJ (E.J.) (EragonJ)](https://github.com/EragonJ) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=e0c33a9dbfe6)

**Pull request merged**